Cockpit 297 (#649)
cockpit-project/cockpit-project.github.io: _posts/2023-07-26-cockpit-297.md

---
title: Cockpit 297
author: skobyda
date: '2023-07-26'
tags: cockpit, podman
slug: cockpit-297
category: release

> summary: 'Administrator can change user shell, Python bridge in fedora-39 and centos-stream-9, Show container's latest checkpoint'

Author: ```suggestion
summary: 'Administrator can change user shell, Python bridge in Fedora 38 and CentOS Stream/RHEL 9, show container's latest checkpoint'
```

Author: Applied, and also fixed the quoting.

> summary: "Administrator can change user shell, Python bridge in Fedora 38 and CentOS Stream/RHEL 9, Show container's latest checkpoint"

Reviewer: ```suggestion
summary: "Support for changing user shell, Python bridge in Fedora 38 and CentOS Stream/RHEL 9, Show container's latest checkpoint"
```

---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
We release regularly.

Here are the release notes from Cockpit 297 and cockpit-podman 73:

> ## Allow administrators to change the user shell

Reviewer: ```suggestion
## Support for changing user shells
```

Author: Please also prefix with "Users:"

> Administrators can now change a users shell via the user account page.

Reviewer: ```suggestion
Administrators can now change user default shells on the Accounts page.
```

![screenshot of ## allow administrators to change the user shell]({{ site.baseurl }}/images/297-allow-administrators-to-change-the-user-shell.png)

![screenshot of ## allow administrators to change the user shell]({{ site.baseurl }}/images/297-allow-administrators-to-change-the-user-shell-2.png)

> ## Move to Python bridge in Fedora 38 and CentOS Stream 9

Reviewer: ```suggestion
## Use the Python bridge in Fedora 38 and CentOS Stream 9
```

> The rewritten Python bridge that was introduced in [Cockpit 294](https://cockpit-project.org/blog/cockpit-294.html) gets enabled on Fedora 38 and CentOS Stream 9. Please [let us know](https://github.com/cockpit-project/cockpit/issues/new?template=bug_report.yml) if you run into any trouble!

Reviewer: ```suggestion
The rewritten Python bridge, introduced in [Cockpit 294](https://cockpit-project.org/blog/cockpit-294.html), is now used on Fedora 38 and CentOS Stream 9. Please [let us know](https://github.com/cockpit-project/cockpit/issues/new?template=bug_report.yml) if you run into any trouble!
```

> ## Podman: Show time of container's latest checkpoint

Reviewer: ```suggestion
## Podman: Show the time of the latest container checkpoint
```
